test/tests/neml_benchmark/linear_strain_hardening/elas_plas_LSH.i

```diff
@@ -0,0 +1,232 @@
+#
+[GlobalParams]
+  order = FIRST
+  family = LAGRANGE
+  volumetric_locking_correction = true
+  displacements = 'disp_x disp_y'
+[]
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 1
+  ny = 1
+  xmax = 1.0
+  ymax = 1.0
+  elem_type = QUAD4
+[]
+
+[Variables]
+  [./disp_x]
+  [../]
+  [./disp_y]
+  [../]
+[]
+
+[AuxVariables]
+  [./eff_plastic_strain]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+[]
+
+[Modules/TensorMechanics/Master]
+  [./all]
+    strain = FINITE
```

Comment:
See if setting this to SMALL still matches

Reply:
Tried this and it didn't work. Do I need to make other changes as well?